appengine-java8/sendgrid/pom.xml

```diff
@@ -1,5 +1,5 @@
 <!--
-Copyright 2015 Google Inc.
+Copyright 2018 Google Inc.
```

Comment:
If updating the copyright, please also change `Inc.` to `LLC` (in other files as well)

Reply:
done.

Reply:
Kokoro is set to look for Google Inc. not Google LLC.